content/en/docs/Writing policies/exceptions.md

```diff
@@ -5,17 +5,13 @@ description: >
 weight: 80
 ---
 
-{{% alert title="Warning" color="warning" %}}
-Policy exceptions are a **beta** feature and requires setting certain container flags to enable. It is not ready for production usage and there may be breaking changes. Normal semantic versioning and compatibility rules will not apply.
-{{% /alert %}}
```

Comment:
So we're proposing that Policy Exceptions will now graduate to stable? Is this backed up by an API change or is it still at beta? Has this discussion to stable been raised with the maintainers?

Reply:
I removed it because exceptions are enabled by default, and there would be no breaking changes.